### ISSUE TITLE
style_demo.ipynb文件，这些写死的模型的路径，window本地跑起来直接报错

### ISSUE THREAD

**Issue by jiehwa** [TencentARC/PhotoMaker#70]
![image](https://github.com/TencentARC/PhotoMaker/assets/3799526/0c7f85a4-f15b-4212-b083-4b90801b14e4)

![image](https://github.com/TencentARC/PhotoMaker/assets/3799526/4245f7fa-1f90-4689-988a-adbbc84e8185)

**Comment by machineminded** [comment]
Yeah - I was just about to open an issue about this.  It appears that `PhotoMakerStableDiffusionXLPipeline` does not support a local `safetensors` format.  We would have to convert the model to diffusers I think.  However, the diffusers `StableDiffusionXLPipeline` does support the `safetensors` format, so not sure what is going on here.

**Comment by Paper99** [comment]
Download corresponding `safetensors` file to `./civitai_models`
